### PR TITLE
Added support for the AkemiPad.

### PR DESCRIPTION
**Pull request by ADPenrose**
## Description

Added support for the AkemiPad: a numpad powered by a ProMicro, with multiple layout options and support for an encoder, a piezo buzzer, an OLED, and per-key RGB lighting.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* As this is the first PR regarding the kintsugi keyboard, no issue has been fixed or closed yet.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/adpenrose/akemipad/keymaps/oled_bongocat/keymap.c
Comment (drashna) [inline keyboards/adpenrose/akemipad/keymaps/oled_bongocat/keymap.c:171]: Could simplify this:
```c
    oled_write_P(PSTR(">0"), false);
    oled_write_ln(get_u8_str(get_current_wpm(), '0'), false);
```

It comes out to the same thing effectively, but there is already code in the core to handle this, basically.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/keymaps/oled_bongocat/keymap.c:171]: Ok, I'll change and test that ASAP.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/keymaps/oled_bongocat/keymap.c]: Done. I played around a bit with the code because it started rendering the first character of the default font (the one that looks like a sad face in https://joric.github.io/qle/). I couldn't fix it, so I just removed the first zero and centered the WPM counter.

File: keyboards/adpenrose/akemipad/keymaps/via/readme.md
Comment (drashna) [inline keyboards/adpenrose/akemipad/keymaps/via/readme.md:16]: No html, please.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/adpenrose/akemipad/keymaps/via/readme.md
Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/keymaps/via/readme.md:16]: Sure, I'll also remove that ASAP.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/keymaps/via/readme.md:16]: Done.

**Review by drashna — APPROVED** [review]

**Comment by ADPenrose** [comment]
Do I need to add another change for this PR to be merged?

**Comment by drashna** [comment]
Mostly, it just needs a second review.

**Comment by ADPenrose** [comment]
Thanks for the answer! Then I'll just wait hahaha.

File: keyboards/adpenrose/akemipad/rules.mk
Comment (fauxpark) [inline keyboards/adpenrose/akemipad/rules.mk:8]:
```suggestion
#   change yes to no to disable
```

File: keyboards/adpenrose/akemipad/rules.mk
Comment (fauxpark) [inline keyboards/adpenrose/akemipad/rules.mk:21]:
```suggestion
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = yes          # Audio output
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = WS2812
ENCODER_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ADPenrose** [comment]
Were the added changes sufficient?

**Comment by ADPenrose** [comment]
I don't know if I found a bug or something, but the QMK CI build keeps failing, while I can compile the code without a problem on my computer.

**Review by fauxpark — APPROVED** [review]

File: keyboards/adpenrose/akemipad/config.h
Comment (fauxpark) [inline keyboards/adpenrose/akemipad/config.h:45]:
```suggestion
#define RGB_MATRIX_LED_COUNT 27
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/adpenrose/akemipad/config.h
Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/config.h:45]: Changing that line is what breaks everything. Moreover, the error message says that the problem is the undeclared DRIVER_LED_TOTAL on some quantum file. Heres an image of the error: https://imgur.com/a/sSemXZ5

Am I doing something wrong?

Comment (fauxpark) [reply keyboards/adpenrose/akemipad/config.h:45]: Your branch is out of date, it will work when merged into current master. Commit and push and the CI build should pass.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/config.h:45]: Ok, thanks for the heads up. I'll do it ASAP.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/config.h]: Ready!

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/config.h]: Hmmm, seems that I need to change some other things. I'll change those too ASAP.

Comment (ADPenrose) [reply keyboards/adpenrose/akemipad/config.h]: There, everything's working as expected!

**Comment by ADPenrose** [comment]
Do I need to change anything else?

**Comment by ADPenrose** [comment]
https://i.imgflip.com/75k2vf.jpg

**Pull request merged**
